### PR TITLE
Add benchmark metric test for UpDownCounter

### PR DESCRIPTION
**Pull request by hanyuancheung**
- Add benchmark metric test for UpDownCounter in `sdk/metric`

### CODE REVIEW

File: sdk/metric/benchmark_test.go
Comment (dmathieu) [reply sdk/metric/benchmark_test.go]: Sorry, there was a typo in my previous comment. I meant should it be skipped (not shipped) instead of being fully commented.
Fully commented test cases can be easy to forget. While a pending/skipped test case remains visible.

Comment (hanyuancheung) [reply sdk/metric/benchmark_test.go]: Yeah I understood, but it could just be commented because of the lack of `global` dependency. I think it could be fixed after the global package is ready in `sdk/metric`.

**Comment by hanyuancheung** [comment]
PTAL, thanks!

**Comment by hanyuancheung** [comment]
Hi guys, PTAL at this PR. Could it be ready to merge?

**Review by hickeyma — CHANGES_REQUESTED** [review]
Thanks for the PR @hanyuancheung. Potential change on `iUpDownCounter` function.

File: sdk/metric/benchmark_test.go
Comment (hanyuancheung) [reply sdk/metric/benchmark_test.go]: Thanks for the suggestion, I've fixed it! PTAL again.

**Review by hickeyma — APPROVED** [review]
LGTM, thanks @hanyuancheung

**Comment by hanyuancheung** [comment]
@MrAlias this pr is ready for merge, ptal.

**Comment by MrAlias** [comment]
@hanyuancheung this PR needs another approval from an approver of the project before it is ready to be merged.

I would also, consider closing this until the new metric SDK is complete. It is going to be work we throwaway and eventually recreate. Maybe instead create an issue to track the future work?

**Comment by hanyuancheung** [comment]
> @hanyuancheung this PR needs another approval from an approver of the project before it is ready to be merged.
> 
> I would also, consider closing this until the new metric SDK is complete. It is going to be work we throwaway and eventually recreate. Maybe instead create an issue to track the future work?

Yeal of course, you could close it when it's ready to release the new metric SDK. After that, I'll open an issue to track this work, and eventually recreate the PR.

**Review by MadVikingGod — APPROVED** [review]

**Comment by hanyuancheung** [comment]
@MrAlias @MadVikingGod pls help to merge this PR. Thanks.

**Pull request merged**
